samples/batch/transcription-enabled-storage/FetchTranscription/TranscriptionProcessor.cs

```diff
@@ -67,14 +67,14 @@ public static async Task ProcessTranscriptionJobAsync(TranscriptionStartedMessag
             }
             catch (WebException e)
             {
-                if (e.Response != null && BatchClient.IsThrottledOrTimeoutStatusCode(((HttpWebResponse)e.Response).StatusCode))
+                if (e.Response != null && (BatchClient.IsThrottledOrTimeoutStatusCode(((HttpWebResponse)e.Response).StatusCode) || ((HttpWebResponse)e.Response).StatusCode == HttpStatusCode.InternalServerError))
```

Comment:
Do you need any backoff time if it is Internal Server error?

Reply:
we got backoff time already in there since we exponentially retry in that case here